aw_watcher_window/macos.py

```diff
@@ -9,9 +9,38 @@ def getInfo() -> str:
     return str(p.stdout, "utf8").strip()
 
 
-def getApp(info) -> str:
+def getApp(info: str) -> str:
     return info.split('","')[0][1:]
 
 
-def getTitle(info) -> str:
+def getTitle(info: str) -> str:
     return info.split('","')[1][:-1]
+
+
+def background_ensure_permissions() -> None:
+    from multiprocessing import Process
+    permission_process = Process(target=ensure_permissions, args=(()))
+    permission_process.start()
+    permission_process.join()
```

Comment:
This is where you block execution!

Reply:
Oh right of course thats what .join() is meant to do. Whoops. Fixed in b829e04